message_ix/tests/data/generate_nightly_test_values.py

```diff
@@ -0,0 +1,28 @@
+import traceback
+
+import ixmp
+import yaml
+
+import message_ix
+
+with open("./scenarios.yaml", mode="r") as f:
+    d = yaml.safe_load(f)
+
+mp = ixmp.Platform("ixmp_dev")
+
+for name, data in d.items():
```

Comment:
Please move this code into the existing file nightly.py:
https://github.com/iiasa/message_ix/blob/afefe8577b3decc16e1446ca3fc41efa72a10e4e/message_ix/testing/nightly.py#L105-L190

You can create a new subcommand of the "message-ix nightly" CLI command, e.g.:

```py
@cli.command("generate-values")
@click.pass_context
def generate_values(context):
   …
```